staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -2878,9 +2891,9 @@ func testRetryWithRateLimiterBackoffAndMetrics(t *testing.T, key string, doFunc
 		// backoff.Sleep before retry 1 (B)
 		(2 * time.Minute).String(),
 	}
-	statusCodesWant := []string{
-		"500",
-		"200",
+	metricStatusCodesWant := []string{
+		"500", "500", // we are counting retries into both metrics.
```

Comment:
this means is the same 500, but the interceptor in the test put them in the same array, right?

Reply:
yes